src/EditorFeatures/CSharp/AutomaticCompletion/AutomaticLineEnderCommandHandler_Helpers.cs

```diff
@@ -748,6 +748,7 @@ private static bool ShouldRemoveBraceForObjectCreationExpression(ObjectCreationE
         // event EventHandler Bar {add; remove;}
         private static bool ShouldRemoveBraceForAccessorDeclaration(AccessorDeclarationSyntax accessorDeclarationNode, int caretPosition)
             => accessorDeclarationNode.Body != null
+               && accessorDeclarationNode.Body.Statements.IsEmpty()
                && accessorDeclarationNode.ExpressionBody == null
                && accessorDeclarationNode.Parent != null
```

Comment:
technically, this check isn't necessary.  teh only thing that ever has a null parent is structured trivia or a compilation unit.

Reply:
:) I was just always trying to avoid use the '!' operator to suppress the null check. But indeed its parent is not null here. Given this PR is probably going to QB I feel it is better to keep the check here. I could open a PR targeting master to remove the check here